packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -609,6 +609,7 @@ For a request to be a _polling_ request it must meet the following request param
 
 Polling requests to the `/api/v1/logs` API have the following semantics:
   - They return every event that occurs in your organization.
+  - They are time-filtered by their internal "persistence time" to avoid skipping records due to system delays (unlike [Bounded Requests](#bounded-requests)).
```

Comment:
Nit: consider replacing "They" with "Events" for clarity/differentiation from requests.